diff -Nru amavis-0.2.1/configure.in amavis-0.2.1.new/configure.in
--- amavis-0.2.1/configure.in	Tue Oct 31 17:19:39 2000
+++ amavis-0.2.1.new/configure.in	Thu Mar  8 22:25:01 2001
@@ -424,7 +424,7 @@
 
 
 AC_CHECKING([*** Kaspersky Lab AntiViral Toolkit Pro (AVP) ***])
-AC_PATH_PROGS(avp, AvpLinux, , $PATH:/usr/bin:/usr/local/bin:/usr/local/AvpLinux)
+AC_PATH_PROGS(avp, AvpLinux, , $PATH:/usr/bin:/usr/local/bin:/usr/local/AvpLinux:/opt/AVP)
 if test "x$avp" = "x"; then
   AC_MSG_WARN([************************************************************])
   AC_MSG_WARN([*** Kaspersky Lab AntiViral Toolkit Pro (AVP) NOT found  ***])
@@ -433,7 +433,7 @@
 fi
 
 AC_CHECKING([*** Kaspersky Lab AVPDaemon ***])
-AC_PATH_PROGS(avpd, AvpDaemon, , $PATH:/usr/bin:/usr/local/bin:/usr/local/AvpLinux)
+AC_PATH_PROGS(avpd, AvpDaemon, , $PATH:/usr/bin:/usr/local/bin:/usr/local/AvpLinux:/opt/AVP)
 if test "x$avpd" = "x"; then
   AC_MSG_WARN([************************************************************])
   AC_MSG_WARN([***           Kaspersky Lab AVPDaemon NOT found          ***])
@@ -442,7 +442,7 @@
 fi
 
 AC_CHECKING([*** Kaspersky AVPDaemonClient ***])
-AC_PATH_PROGS(avpdc, AvpDaemonClient, , $PATH:/usr/bin:/usr/local/bin:/usr/local/AvpLinux)
+AC_PATH_PROGS(avpdc, AvpDaemonClient, , $PATH:/usr/bin:/usr/local/bin:/usr/local/AvpLinux:/opt/AVP)
 if test "x$avpdc" = "x"; then
   AC_MSG_WARN([************************************************************])
   AC_MSG_WARN([***        Kaspersky Lab AVPDaemonClient NOT found       ***])
@@ -463,7 +463,7 @@
 
 if test "x$avpdc" = "x" ; then
  AC_CHECKING([*** Kaspersky AVPDaemonClient ***])
- AC_PATH_PROGS(avpdc, AvpDaemonTst, , $PATH:/usr/bin:/usr/local/bin:/usr/local/AvpLinux)
+ AC_PATH_PROGS(avpdc, AvpDaemonTst, , $PATH:/usr/bin:/usr/local/bin:/usr/local/AvpLinux:/opt/AVP)
  if test "x$avpdc" = "x"; then
    AC_MSG_WARN([************************************************************])
    AC_MSG_WARN([***        Kaspersky Lab AVPDaemonTst NOT found          ***])
